tests/hooks/test_dbapi_hook.py

```diff
@@ -77,8 +77,8 @@ def test_get_records_exception(self):
         with self.assertRaises(RuntimeError):
             self.db_hook.get_records(statement)
 
-        self.conn.close.call_count == 1
-        self.cur.close.call_count == 1
+        assert self.conn.close.call_count == 1
+        assert self.cur.close.call_count == 1
```

Comment:
https://github.com/apache/airflow/pull/10842/files#diff-a7a21a33feeecc751f576d01ec2bd1d0L108-L109 -> other tests in this file contains assert properly